src/lib/expansion/expansion-panel.scss

```diff
@@ -26,12 +26,12 @@
   margin: 16px 0;
 
   .mat-accordion > &:first-child,
-  .mat-accordion > *:first-child & {
+  .mat-accordion > *:first-child:not(.mat-expansion-panel) & {
```

Comment:
@josephperrott this selector was introduced in 85252b3ea602f6963acb8916a2c44c27e5d91911, but it wasn't clear from the original PR exactly what it was fixing. These changes shouldn't break anything, as far as I could tell, but it might be worth double-checking.

Reply:
The intention is to allow for intermediate elements in accordions, to allow for components to be used to create expansion panels


So it would end up like this:
```
<mat-accordion>
  <my-comp>
    <mat-expansion-panel>
...
```

I think that this should be the correction needed, though it might be good for us to explore controlling this spacing with an angular animation.  But I can do that in another PR.